### DISCUSSION TITLE
Inversion symmetry about the midpoint of a bond

### DISCUSSION THREAD

**Discussion by Sreema1996** [QuSpin/QuSpin#546] [Q&A] [answered]
Hi Developers,

I'm currently studying a one-dimensional spinless fermionic model which has inversion symmetry about the midpoint of a bond. Is there any way to implement this in the QuSpin code? Could you please help me regarding this?
Thanks in advance!

**Comment by mgbukov** [comment]
Can you provide us with a minimal version of the Hamiltonian that has this symmetry?

**Comment by Sreema1996** [comment]
@mgbukov,

Thanks for your reply!

My model is the one-dimensional t-V model with staggered onsite potential. I initially thought the model was symmetric under i->N-i-1 transformation. (i is site no. and N is the total system size). But I was incorrect in thinking that since the staggered potential breaks it. However, I found another symmetry transformation, i->N-i and I believe my model is symmetric under this second transformation. If one considers a ring geometry, i->N-i is like a reflection about the line connecting the zeroth and N/2th site. Could you please tell me how to implement this symmetry in my QuSpin code?

**Comment by mgbukov** [comment, marked as answer]
I think this should be doable using quspin's [general bases](https://weinbe58.github.io/QuSpin/generated/quspin.basis.spin_basis_general.html#quspin.basis.spin_basis_general); there u have the liberty to define your own transformation by telling it which sites to permute. 

To check if the symmetry exists, try computing the spectrum (1) with and (2) without the symmetry on. The former should be a subset of the latter.

**Comment by Sreema1996** [comment]
@mgbukov,

Thank you for your suggestion!
It worked nicely.
